operator-pipeline-images/operatorcert/static_tests/common/bundle.py

```diff
@@ -161,3 +165,26 @@ def check_bundle_release_config(bundle: Bundle) -> Iterator[CheckResult]:
             yield from _check_semver_allowed_channels(
                 template_names_in_mapping[template["template_name"]], template
             )
+
+
+def validate_schema_bundle_release_config(bundle: Bundle) -> Iterator[CheckResult]:
+    """
+    Validate the bundle release config against the json schema
+    """
+    if not bundle.release_config:
+        # missing release config (this is assumed to be ok)
+        return
+    # (assumes json schema lives in same dir as this python script)
+    path_me = os.path.dirname(os.path.abspath(__file__))
+    path_schema = os.path.join(path_me, "release-config-schema.json")
```

Comment:
Can you please move the schema file outside of the code of python submodule? A good location might be `operatorcert/schemas/release-config-schema.json` since we will have multiple schema files.

Reply:
Agreed, that makes better sense, updated